README.md

```diff
@@ -1,6 +1,32 @@
 # Description
 
-A collection of salt states used to provision a kubernetes cluster.
+A collection of Salt states used to provision a kubernetes cluster
+in the context of CaaSP/MicroOS.
+
+# Salt states and CaaSP architecture
+
+The Salt state creates an architecture that depends on some (already existing) elements. This is the expected setup before running the orchestration:
+
+![](docs/k8s-before-orchestration.svg)
+
+The _salt master_ is installed in the _Controller Node_ and is reponsible for
```

Comment:
I believe the naming is "Administration Dashboard" instead of "Controller Node".